CONTRIBUTING.md

```diff
@@ -206,6 +206,13 @@ rustup toolchain link stage2 build/x86_64-unknown-linux-gnu/stage2
 rustup override set stage2
 ```
 
+Important: You need to delete the miri cache (located at `~/.cache/miri` on Linux; the exact location is printed after the library build: "A libstd for Miri is now available in ...") when
+you change the stdlib, otherwise the old, chached version will be used.
```

Comment:
```suggestion
Important: You need to delete the Miri cache when you change the stdlib; otherwise the
old, chached version will be used. On Linux, the cache is located at `~/.cache/miri`; the exact
location is printed after the library build: "A libstd for Miri is now available in ...".
```